qtpy/Qt3DAnimation.py

```diff
@@ -4,18 +4,36 @@
 # Licensed under the terms of the MIT License
 # (see LICENSE.txt for details)
 # -----------------------------------------------------------------------------
+
 """Provides Qt3DAnimation classes and functions."""
 
-# Local imports
-from . import PYQT5, PYSIDE2, PythonQtError, PYSIDE_VERSION
+from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, QtBindingsNotFoundError, QtModuleNotInstalledError
```

Comment:
Please make sure that lines preserve the 79 column width we use in the rest of our code.